Source/Particles/MultiParticleContainer.H

```diff
@@ -22,6 +22,11 @@
 // concrete class derived from WarpXParticlContainer.
 //
 
+#ifdef WARPX_QED
```

Comment:
breit_wheeler_engine_wrapper and quantum_sync_engine_wrapper are wrappers for the engines developed in PICSAR. During the simulation run we will use only calls to static functions of these classes. However, somewhere in the code, we'll need to initialize or load some lookup tables. Which is the most appropriate part of the code to do that?

Reply:
Once in a simulation. So MultiParticleContainer::InitData is definitively the most appropriate place in the code. I'll do that (probably) in the next PR